public/app/features/explore/Logs/Logs.tsx

```diff
@@ -556,7 +577,7 @@ class UnthemedLogs extends PureComponent<Props, State> {
             config.featureToggles.logsExploreTableVisualisation
               ? this.state.visualisationType === 'logs'
                 ? 'Logs'
-                : 'Table'
+                : 'Table (experimental)'
```

Comment:
What do you think about using a `Tag` component to indicate this? Like the `beta` tag that was used in the query builder: https://github.com/grafana/grafana/pull/55150

Reply:
PanelChrome title only takes a string so we'd need to tweak things a bit to get the tag in there.

Reply:
I think you can use `titleItems` https://developers.grafana.com/ui/latest/index.html?path=/docs/visualizations-panelchrome--basic.